Terminal.Gui/Views/DateField.cs

```diff
@@ -48,8 +41,7 @@ public class DateField : TextField {
 	/// <param name="x">The x coordinate.</param>
 	/// <param name="y">The y coordinate.</param>
 	/// <param name="date">Initial date contents.</param>
-	/// <param name="isShort">If true, shows only two digits for the year.</param>
-	public DateField (int x, int y, DateTime date, bool isShort = false) : base (x, y, isShort ? 10 : 12, "") => SetInitialProperties (date, isShort);
+	public DateField (int x, int y, DateTime date) : base (x, y, 12, "") => SetInitialProperties (date);
```

Comment:
While you're in here, would you mind removing these constructors with parameters? See ##3171.

Reply:
Done.